### PR TITLE
Using `importlib.metadata` to locate `extra-dll` on Windows.

### LINKED ISSUE #206: PyPI Windows wheel installs unimportable package
After pip-installing bezier from PyPI on Windows, the package cannot be imported:
```
~> python -c "import bezier"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "...\Miniconda3\envs\default\lib\site-packages\bezier\__init__.py", line 44, in <module>     
    __config__.handle_import_error(exc, "_speedup")
  File "...\Miniconda3\envs\default\lib\site-packages\bezier\__config__.py", line 92, in handle_impo
rt_error
    raise caught_exc
  File "...\Miniconda3\envs\default\lib\site-packages\bezier\__init__.py", line 40, in <module>     
    import bezier._speedup  # noqa: F401
ImportError: DLL load failed while importing _speedup: The specified module could not be found.
```

This is in a Py3.8 conda env.

### PR DESCRIPTION
**Pull request by dhermes**
Fixes #206.

/cc @anntzer

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -63,7 +63,8 @@
 DLL_HASH_ENV = "BEZIER_DLL_HASH"
 REQUIREMENTS = ("numpy >= 1.18.1",)
 EXTRAS_REQUIRE = {
-    "full": ["matplotlib >= 3.0.0", "scipy >= 1.4.1", "sympy >= 1.5.1"]
+    "full": ["matplotlib >= 3.0.0", "scipy >= 1.4.1", "sympy >= 1.5.1"],
+    ':python_version<"3.8"': ["importlib-metadata"],
```
Comment (dhermes) [inline setup.py]: I should probably also limit this to just Windows.

Comment (dhermes) [reply setup.py]: Probably should add a reference to [PEP 508][1] and/or [PEP 496][2], I always forget what they say.

The check `platform_system == "Windows"` "works best" here (or `os_name == "nt"`).

[1]: https://www.python.org/dev/peps/pep-0508/
[2]: https://www.python.org/dev/peps/pep-0496/

**Comment by anntzer** [comment]
Thanks for the quick fix!

**Comment by anntzer** [comment]
Any chance for a new release with this fix?  Thanks!

**Comment by dhermes** [comment]
@anntzer I'll try to get one out in the next week or so. Trying to squeeze in some module moves into `bezier.hazmat` right now.

**Comment by anntzer** [comment]
No worries, thanks.

**Comment by dhermes** [comment]
@anntzer I just cut a release: https://pypi.org/project/bezier/2020.5.19/

**Comment by travisbriles** [comment]
Is anyone else still seeing this error with version 2020.5.19 on windows 10 + Anaconda3?  I installed with 'pip install bezier'.  I was able to get this work with my Mac OS Catalina.  

bezier error

**Comment by dhermes** [comment]
@travisbriles This has been reported in #237. I'll update my comments there with a second suggestion (there is an existing suggestion about installing without the Fortran speedups https://github.com/dhermes/bezier/issues/237#issuecomment-651317889).

**Comment by travisbriles** [comment]
Thanks for the reply!  I hadn't seen #237 yet.

**Pull request merged**